app/controllers/maps_controller.rb

```diff
@@ -1,10 +1,23 @@
+# frozen_string_literal: true
+
 class MapsController < ApplicationController
+  def show
+    locations = Location.where(map_id: params[:id])
 
-  def index
-    puts "dd"
-  end
+    raise ActiveRecord::RecordNotFound if locations.empty?
 
-  def show
-    puts "dddddd"
+    markers = LocationServices::LocationServicesGeoJson
+              .new(Location.where(map_id: params[:id]))
```

Comment:
`.new(Location.where(map_id: params[:id]))`

Probably intended to write `.new(locations)` here, as defined above

Reply:
Yes, forgot to change that. Thanks!